static/usage/v7/input/set-focus/react.md

```diff
@@ -0,0 +1,23 @@
+```tsx
+import React, { useRef } from 'react';
+import { IonInput, IonItem, IonList, IonButton, IonPage } from '@ionic/react';
+
+const Home = () => {
+  const input = useRef<HTMLIonInputElement>(null);
+
+  return (
+    <IonPage>
+      <IonList>
+        <IonItem>
+          <IonButton onClick={() => input.current?.setFocus()}>Click to set focus</IonButton>
+        </IonItem>
+        <IonItem>
+          <IonInput ref={input} label="Email" labelPlacement="floating"></IonInput>
+        </IonItem>
+      </IonList>
+    </IonPage>
+  );
+};
+
+export default Home;
+```
```

Comment:
IonPage is no longer needed now that we removed routing from this demo:

```suggestion
```tsx
import React, { useRef } from 'react';
import { IonInput, IonItem, IonList, IonButton } from '@ionic/react';

const Home = () => {
  const input = useRef<HTMLIonInputElement>(null);

  return (
    <IonList>
      <IonItem>
        <IonButton onClick={() => input.current?.setFocus()}>Click to set focus</IonButton>
      </IonItem>
      <IonItem>
        <IonInput ref={input} label="Email" labelPlacement="floating"></IonInput>
      </IonItem>
    </IonList>
  );
};

export default Home;
```
```